### PR TITLE
Made the changes according to review comments-QA-88

### PR DESCRIPTION
**Pull request by vardhan1991**

### CODE REVIEW

File: src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java [hunk 1]
```diff
@@ -79,9 +79,19 @@ public Object execute() throws ApiException {
...
+        for (int idx = 0; idx < keyIds.size(); idx++){
             try {
-                this.securityObjectsApi.deleteSecurityObject(keyId);
+                this.securityObjectsApi.deleteSecurityObject(keyIds.get(idx));
+
+                /* Adding a logging statement that prints the key being deleted, for every 100 keys -
+                   to give the User a sense of progress */
+
+                if(idx % 100 == 0)
+                    LOGGER.log(Level.INFO, "Deleted key : " + keyIds.get(idx));
```
Comment (manasag) [inline src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java:93]: Better message would be: "X keys deleted", X being idx.

Comment (vardhan1991) [reply src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java:93]: When making this change, we ran into the issue where each thread started to print its own version of the idx counter, resulting in logs like this:

[INFO] Apr 14, 2020 2:20:23 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 0 keys!
[INFO] Apr 14, 2020 2:20:52 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 100 keys!
[INFO] Apr 14, 2020 2:21:21 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 200 keys!
[INFO] Apr 14, 2020 2:21:51 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 300 keys!
[INFO] Apr 14, 2020 2:22:22 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 0 keys!
[INFO] Apr 14, 2020 2:22:51 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 100 keys!
[INFO] Apr 14, 2020 2:23:21 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 200 keys!
[INFO] Apr 14, 2020 2:23:50 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 300 keys!

Where we are resetting to 0 once the previous thread completes deleting all the keys created during its lifecycle.

Comment (vardhan1991) [reply src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java:93]: Should we print a static "Deleted 100 keys" text to work around this? Borrowing the count from the previous thread seemed to be overkill for this feature.

Comment (manasag) [reply src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java:93]: I agree, but printing random keyId being deleted also doesn't provide any useful info. We can get thread id for current thread and print as "Thread id <id>: 100 keys deleted". Now the messages are not ambiguous.

Comment (vardhan1991) [reply src/main/java/com/fortanix/sdkms/performance/sampler/KeyGenerationSampler.java]: The logger statement has been modified to read like this:

[INFO] INFO: Deleted 100 keys in thread number: 3
[INFO] Apr 22, 2020 2:39:14 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 100 keys in thread number: 1
[INFO] Apr 22, 2020 2:42:08 PM com.fortanix.sdkms.performance.sampler.KeyGenerationSampler teardownTest
[INFO] INFO: Deleted 100 keys in thread number: 2

**Pull request merged**